pam/integration-tests/ssh_test.go

```diff
@@ -39,6 +40,21 @@ var (
 	sshHostPortRegex     *regexp.Regexp
 
 	sshDefaultFinalWaitTimeout = sleepDuration(3 * defaultSleepValues[authdWaitDefault])
+
+	prepareSSHTestsOnce sync.Once
+	sshTestsPrepared    bool
```

Comment:
Shouldn't this be atomic? Or mutex-protected?

Reply:
> now potentially we may still enter this stanza twice

that was on purpose, so that the "Waiting for SSH tests to be prepared" log message is printed before we start waiting for the operation even when it's done by another goroutine